nnmnkwii/paramgen/_mlpg.py

```diff
@@ -334,14 +349,31 @@ def unit_variance_mlpg_matrix(windows, T):
     """
     win_mats = build_win_mats(windows, T)
     sdw = np.max([win_mat.l + win_mat.u for win_mat in win_mats])
+    max_win_width = np.max([max(win_mat.l, win_mat.u) for win_mat in win_mats])
 
     P = bm.zeros(sdw, sdw, T)
+
+    # set edge precitions to zero
+    precisions = bm.zeros(0, 0, T)
+    precisions.data[:, max_win_width:-max_win_width] += 1.0
+
+    mod_win_mats = list()
```

Comment:
```suggestion
    mod_win_mats = []
```

`[]` is more pythonic and seems faster. https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better